### PR TITLE
Fixes https://github.com/SiCKRAGETV/sickrage-issues/issues/3010

### PR DESCRIPTION
**Pull request by OmgImAlexis**
https://github.com/SiCKRAGETV/sickrage-issues/issues/3010

### CODE REVIEW

**Comment by SiCKRAGEB0T** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Give a description on what the PR is for.
- [x] Make sure you send a PR against the DEVELOP branch

Please make sure you also read [contribution guide](https://github.com/SiCKRAGETV/SickRage/blob/master/contributing.md) and followed all the steps.

Thanks!

**Comment by fernandog** [comment]
@OmgImAlexis the title of the popup window is wrong when browsing for restore.

File: gui/slick/js/new/config_backuprestore.js [hunk 1]
```diff
@@ -1,5 +1,119 @@
-$(document).load(function(){
+$(document).ready(function(){
     $('#backupDir').fileBrowser({ title: 'Select backup folder to save to', key: 'backupPath' });
     $('#backupFile').fileBrowser({ title: 'Select backup files to restore', key: 'backupFile', includeFiles: 1 });
```
Comment (OmgImAlexis) [inline gui/slick/js/new/config_backuprestore.js]: @fernandog I just copied the code from the old file, check line 2 and 3.

Comment (fernandog) [inline gui/slick/js/new/config_backuprestore.js]: @OmgImAlexis I see that but still not working. If you click "restore" first after load the page, it shows the restore title. But if then you click backup, it uses the restore message. The other way around too if you refresh page. Don't know how to fix it. Look like browser is saving window settings and not re-creating the window

Comment (OmgImAlexis) [inline gui/slick/js/new/config_backuprestore.js]: Well either way it's an old bug and not something that's super important at the moment, open an issue and I'll look into it.

**Pull request merged**